### PR TITLE
Feature/146683087/get jwt on login

### PR DESCRIPTION
**Pull request by codejockie**
Pull Request Convention

#146683087 Users should receive a JWT on successful login
#146683089 Users should receive a JWT on successful signup

    #### What does this PR do?
    #### This PR changes the authentication method from session auth to JWT auth

    #### Description of Tasks to be completed
    #### Users once authenticated via the login api should be returned a JWT to be required on subsequent call to all other endpoints that require authentication
    #### Users once authenticated via the signup api should be returned a JWT to be required on subsequent call to all other endpoints that require authentication

    #### How should this be manually tested?
    #### N/A

    #### Any background context you want to provide?
    #### No

    #### What are the relevant pivotal tracker stories?
    #### Users should receive a JWT on successful login
    #### Users should receive a JWT on successful signup

    #### Screenshots (if appropriate)
    #### None

    #### Questions:
    #### None

### CODE REVIEW

**Review by tonerdo — APPROVED** [review]

File: public/js/directives.js [hunk 1]
```diff
@@ -71,9 +71,13 @@ angular.module('mean.directives', [])
       link: function(scope, elem, attr) {
         scope.showOptions = true;
 
-        if (scope.$$childHead.global.authenticated === true) {
-          scope.showOptions = false;
-        }
+        // if (scope.$$childHead.global.authenticated === true) {
+        //   scope.showOptions = false;
+        // }
```
Comment (tonerdo) [inline public/js/directives.js]: Don't commit commented out code

File: server.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 /**
  * Module dependencies.
  */
+require('dotenv').config();
```
Comment (tonerdo) [inline server.js:4]: Why? You're not using an `.env` file

**Review by tonerdo — CHANGES_REQUESTED** [review]

File: server.js (hunk 2 above)
Comment (codejockie) [reply server.js:4]: I didn't add that require statement, it was part of the original codebase. for the commented code, I'd fix that.

File: public/js/directives.js (hunk 1 above)
Comment (codejockie) [reply public/js/directives.js]: should I close the PR?

Comment (tonerdo) [reply public/js/directives.js]: @codejockie please don't ask me this question. I've talked with you guys about this before

Comment (codejockie) [reply public/js/directives.js]: I know what to do. Thank you.

File: server.js (hunk 2 above)
Comment (codejockie) [inline server.js]: Done.

**Review by tonerdo — APPROVED** [review]

**Pull request closed without merging**